src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -327,19 +350,59 @@ class RemoteMachineTrainingService implements TrainingService {
         return deferred.promise;
     }
 
-    public cleanUp(): Promise<void> {
+    public async cleanUp(): Promise<void> {
```

Comment:
Add comments to illustrate that cleanUp has a timeout (10 seconds right now? ) So the caller can know its timeout policy.

Reply:
add a comment `cleanup() has a time out of 10s to clean remote connections `